src/components/TokenClaimDialog.tsx

```diff
@@ -319,6 +355,26 @@ function useRewardClaims() {
     },
     [chainId, account, rewardsContract, updateClaimStatus],
   )
+
+  const claimGaugeReward = useCallback(
```

Comment:
Can this be updated to also call `claim_rewards(account)` on the gauge's `LiquidityV5` contract as well, if it has rewards? (`gaugeData.gauges[addr].rewards.length > 0`)